_people/Nils_Vu.markdown

```diff
@@ -0,0 +1,33 @@
+---
+name: Nils Vu
+lastname: Vu
+group: postdocs
+position: Postdoctoral Researcher
+address: Cahill Center for Astronomy and Astrophysics
+institution: California Institute of Technology
+email: nilsvu@caltech.edu
+website: https://nilsvu.de
+specialties: Numerical relativity, black holes, elliptic PDEs, initial data
+---
+
+##### Research profile
+
+I develop numerical relativity simulations for the next generation of
+gravitational wave detectors, such as the
+[LISA space mission](https://www.esa.int/Science_Exploration/Space_Science/LISA).
+I co-lead the development of
+[SpECTRE](https://github.com/sxs-collaboration/spectre), our next-generation
+numerical relativity code that combines discontinuous Galerkin methods with
+massive parallelism.
+
+For details, see my [website](https://nilsvu.de).
+
```

Comment:
Do you want this here, given that you added your `website` field above?

Reply:
Yea let's keep this for now. We can see if we want to standardize these things later. E.g. I think it would also make sense to add a link to ORCID.